### DISCUSSION TITLE
[Question] Connect with TLS

### DISCUSSION THREAD

**Discussion by trajanoreuter** [tchiotludo/akhq#683] [Q&A] [answered]
Hi, we are working with Amazon MSK as our Kafka Cluster and running `akhq` with Kubernetes.
Currently validating the use of the tool by connecting to `PLAINTEXT` on port `9092`.
I was taking a look at the documentation and it was not clear how I would be able to connect via `TLS` on port `9094` of my MSK cluster. Since the documentation leaves a note
`The kafka brokers must be configured with the Strimzi library and an OAuth2 provider (Keycloak example) .`
Is this step also necessary in this scenario?
I'm sorry for creating the issue, I didn't find a forum or something similar.

**Comment by tchiotludo** [comment, marked as answer]
I don't have any amazon account to test but looking [here](https://github.com/aws-samples/amazon-msk-client-authentication), it seems that there is some sample in *Create a client.properties file and put the following in it* , must be like that I think : 
```yaml 
akhq:
  connections:
    ssl-dev:
      properties:
          security.protocol=SSL
          ssl.truststore.location=/tmp/kafka.client.truststore.jks
          ssl.keystore.location=/tmp/kafka.client.keystore.jks
          ssl.keystore.password=<password that you used for the keystore when running the jar file>
          ssl.key.password=<password that you used for the keystore when running the jar file>
          ssl.truststore.password=<truststore password, default = changeit>
```

Just keep in mind you need to pass the same properties as a Java client
